software/open-quartermaster.yml

```diff
@@ -0,0 +1,19 @@
+name: "Open QuarterMaster"
+website_url: "https://openquartermaster.com/"
+source_code_url: "https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster"
+description: "A powerful inventory management system, designed to be flexible and scalable."
+licenses:
+  - GPL-3.0
+platforms:
+  - Java
+  - Python
+  - PHP
```

Comment:
```suggestion
```
Given the only installation method is deb and bash, I would remove these.

Reply:
Interesting, given I wouldn't call these techs installation routes, besides PHP I suppose, I internalized this section as more of a 'built with'. Can remove to confirm that mix up.